components/base/kinematicbase/kinematics.go

```diff
@@ -49,10 +43,22 @@ func WrapWithDifferentialDriveKinematics(
 	if err != nil {
 		return nil, err
 	}
-	if properties.TurningRadiusMeters != 0 {
```

Comment:
[q]: should we add a check to make sure TurningRadiusMeters is not a negative value?